### ISSUE TITLE
Fixes and small things for make-srcset v1

### ISSUE THREAD

**Issue by HHH-GH** [HHH-GH/photo-html#1]
Add to the list, check them off when fixed.

- [x] Trailing period is stripped from alt text and added back as appropriate in output, but when the alt text is displayed in the top form field the period is also stripped but not added back - the period should be included if it was in the original form submission
- [x] Alt text in image templates is in inconsistent location - put it at the end of the image tag always.
- [x] Make the copy-paste parts of the form select contents on focus 

```
<input type="text" value="Some text"
  onfocus="this.select()"
  onmouseup="event.preventDefault()">
```

- [x] did_validate is never set as Y, so the success message does not display. Set it as did_validate = Y at the start? As there are a lot of places where it's set as 'N' throughout the code.
